src/lib/db/feature-toggle-client-store.ts

```diff
@@ -208,15 +209,25 @@ export default class FeatureToggleClientStore
         // strip away unwanted properties
         const cleanedFeatures = features.map(({ strategies, ...rest }) => ({
             ...rest,
-            strategies: strategies?.map(({ id, title, ...strategy }) => ({
-                ...strategy,
-
-                ...(isPlayground && title ? { title } : {}),
-
-                // We should not send strategy IDs from the client API,
-                // as this breaks old versions of the Go SDK (at least).
-                ...(isAdmin || isPlayground ? { id } : {}),
-            })),
+            strategies: strategies
+                ?.sort((strategy1, strategy2) => {
+                    if (
+                        typeof strategy1.sortOrder === 'number' &&
+                        typeof strategy2.sortOrder === 'number'
+                    ) {
+                        return strategy1.sortOrder - strategy2.sortOrder;
+                    }
+                    return 0;
```

Comment:
I feel silly saying this but where do items that end up resolving to 0 actually end up in the final list? I'm guessing this is dependent on the JS engine's implementation of the sort? What if we do something like:

```
strategies.sort((strategy1, strategy2) => {
    const sortOrder1 = typeof strategy1.sortOrder === 'number' ? strategy1.sortOrder : -Infinity;
    const sortOrder2 = typeof strategy2.sortOrder === 'number' ? strategy2.sortOrder : -Infinity;

    return sortOrder1 - sortOrder2;
});
```

I want to say that's both very explicit and deterministic independent of whatever JS feels like doing

Reply:
They way I read it is 0 means don't change the order. I'm afraid of arithmetics with Infinity in JS since we'll get NaN soon

Reply:
Yeah I think it's probably okay, from some reading, most modern JS engines should what we expect here. NaN should also behave like a 0 here too.

This is mostly me nitpicking, I think it's probably okay as is